service/src/main/java/bio/terra/tanagra/service/accesscontrol/impl/AouWorkbenchAccessControl.java

```diff
@@ -65,18 +72,18 @@ public String getDescription() {
   @Override
   public void initialize(List<String> params, String basePath, String oauthClientId) {
     // Store the basePath and oauthClientId first, so we can use it when calling the workbench API.
-    if (basePath == null || oauthClientId == null) {
-      throw new IllegalArgumentException(
-          "Base URL and OAuth client id are required for Workbench API calls");
+    // check: oauthClientId not required?
+    if (basePath == null) {
+      throw new IllegalArgumentException("Base URL is required for Workbench API calls");
     }
     this.basePath = basePath;
-    this.oauthClientId = oauthClientId;
```

Comment:
Could you also please update the `ACCESS_CONTROL.md` doc to remove the oauth client id for the AoU-RW section?

Reply:
done